echopype/tests/test_preprocess.py

```diff
@@ -54,24 +54,90 @@ def test_remove_noise():
     assert np.count_nonzero(null.isel(frequency=0, range_bin=slice(None, 50))) == 6
 
 
+def _construct_MVBS_toy_data(nfreq, npings, nrange, ping_size, range_bin_size):
+    """Construct data with values that increase every ping_num and range_bin_num
+    so that the result of computing MVBS is a smaller array
+    that increases regularly for each resampled ping_time and range_bin
+
+    Parameters
+    ----------
+    nfreq : int
+        number of frequencies
+    npings : int
+        number of pings
+    nrange : int
+        number of range_bins
+    ping_size : int
+        number of pings with the same value
+    range_bin_size : int
+        number of range_bins with the same value
+
+    Returns
+    -------
+    np.ndarray
+        Array with blocks of ping_times and range_bins with the same value,
+        so that computing the MVBS will result in regularly increasing values
+        every row and column
+    """
+    data = np.ones((nfreq, npings, nrange))
+    for p_i, ping in enumerate(range(0, npings, ping_size)):
+        for r_i, rb in enumerate(range(0, nrange, range_bin_size)):
+            data[0, ping:ping + ping_size, rb:rb + range_bin_size] += r_i + p_i
+    # First frequency increases by 1 each row and column, second increases by 2, third by 3, etc.
+    for f in range(nfreq):
+        data[f] = data[0] * (f + 1)
+
+    return data
+
+
+def _construct_MVBS_test_data(nfreq, npings, nrange_bins):
+    """Construct data with values that increases regularly
+    every ping and range_bin
+
+    Parameters
+    ----------
+    nfreq : int
+        number of frequencies
+    npings : int
+        number of pings
+    nrange_bins : int
+        number of range_bins
+
+    Returns
+    -------
+    np.ndarray
+        Array with values that increases regularly
+        every ping and range_bin
+    """
+
+    # Construct test array
+    test_array = np.indices((npings, nrange_bins))
+    test_array = np.add(*test_array)
+    test_array = np.array([test_array] * nfreq) + 1
+
+    # Increase test data by a multiple each frequency
+    for f in range(nfreq):
+        test_array[f] = (test_array[0] * (f + 1))
```

Comment:
Line 116-120 can be done using a list comprehension?
Results of line 116 are effectively not used and overwritten in 120.

It is fine to combine 114-115. As I said in the comments, I put them in separate lines in my code for clarity.